all.json

```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
+		"bnbfaucet.site",
+		"bonowatch.com",
+		"coinfactory.digital",
+		"comicsapesocialsclub.com",
```

Comment:
This looks like just an NFT?

Reply:
comicsapesocialsclub.com appears to be legit, not fake mint, removed